proto/v1alpha1/rode.proto

```diff
@@ -94,6 +94,12 @@ service Rode {
     };
     option (google.api.method_signature) = "filter";
   }
+  rpc ListPolicyVersions(ListPolicyVersionsRequest) returns (ListPolicyVersionsResponse) {
+    option (google.api.http) = {
+      get: "/v1alpha1/policies/{id}/versions"
```

Comment:
this makes me want to re-evaluate my earlier decisions regarding IDs for generic resources + versions.  so much easier to understand what's going on when the endpoint looks like this.

Reply:
I'll have to find the PR, but someone did open one that changed grpc-gateway to work off the raw path. It ended up abandoned, but I think that would allow us to use the resource uri in the path with double percent-encoding